src/main/java/net/minecraftforge/client/ArmorOverlayHandler.java

```diff
@@ -0,0 +1,96 @@
+package net.minecraftforge.client;
+
+import net.minecraft.client.model.ModelBase;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.texture.TextureManager;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.inventory.EntityEquipmentSlot;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.common.ForgeModContainer;
+
+/**
+ * Offshore handler for the Armors' extra passes. Specifically for reducing patch size & maintaining readability.
+ * */
+public class ArmorOverlayHandler
+{
+
+    private static final java.util.Map<net.minecraftforge.fml.common.registry.RegistryDelegate<Item>, IArmorOverlay> armorOverlayMap = com.google.common.collect.Maps.newHashMap();
```

Comment:
Yeah derp moment there. More or less transferred the vanilla method and retrofitted it to match my needs :P Will update the imports